examples/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ include_directories(${CMAKE_SOURCE_DIR}/include)
 include_directories(${CMAKE_SOURCE_DIR})
 
 # Creates a separate device code module for each SYCL* kernel
-# so that kernel for Dg2 and Xe will be JIT separately
+# so that kernel for XeHpg and Xe will be JIT separately
```

Comment:
```suggestion
# so that kernel for XeHpc, XeHpg, and XeLpg will be JIT separately
```

or 

```suggestion
# so that kernel for different device architectures will be JIT separately
```